src/Features/VisualBasic/Portable/CodeRefactorings/MoveType/VisualBasicMoveTypeService.vb

```diff
@@ -13,5 +16,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeRefactorings.MoveType
         <ImportingConstructor>
         Public Sub New()
         End Sub
+
+        Protected Overrides Async Function GetRelevantNodeAsync(document As Document, textSpan As TextSpan, cancellationToken As CancellationToken) As Task(Of TypeBlockSyntax)
```

Comment:
note: can we beef up the VB tests as well.  any new sorts of tests we added for C# we should also add to VB.  That will also be useful in case we want to refactor this in the future.  it will help ensure no unintentional changes in behavior that we'r enot aware of.